2014-02-05T03:11:32-08:00 ERR (3): Application_Model_DbTable_Users::getUserDetail - exception 'PDOException' with message 'SQLSTATE[HY093]: Invalid parameter number: no parameters were bound' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:228
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(228): PDOStatement->execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Users.php(15): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(65): Application_Model_DbTable_Users->getUserDetail(NULL)
#6 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(68): Application_Model_DbTable_Requests->getviewRequest(0)
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->viewrequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('viewrequestActi...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(1): require_once('C:\wamp\www\dev...')
#14 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[HY093]: Invalid parameter number: no parameters were bound' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:234
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#3 C:\wamp\www\dev.ppeditor\application\models\DbTable\Users.php(15): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(65): Application_Model_DbTable_Users->getUserDetail(NULL)
#5 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(68): Application_Model_DbTable_Requests->getviewRequest(0)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->viewrequestAction()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('viewrequestActi...')
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#9 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#11 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#12 C:\wamp\www\dev.ppeditor\index.php(1): require_once('C:\wamp\www\dev...')
#13 {main}
2014-02-05T03:40:43-08:00 ERR (3): Application_Model_DbTable_Users::getUserDetail - exception 'PDOException' with message 'SQLSTATE[HY093]: Invalid parameter number: no parameters were bound' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:228
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(228): PDOStatement->execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Users.php(15): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(65): Application_Model_DbTable_Users->getUserDetail(NULL)
#6 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(68): Application_Model_DbTable_Requests->getviewRequest(0)
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->viewrequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('viewrequestActi...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(1): require_once('C:\wamp\www\dev...')
#14 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[HY093]: Invalid parameter number: no parameters were bound' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:234
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#3 C:\wamp\www\dev.ppeditor\application\models\DbTable\Users.php(15): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(65): Application_Model_DbTable_Users->getUserDetail(NULL)
#5 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(68): Application_Model_DbTable_Requests->getviewRequest(0)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->viewrequestAction()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('viewrequestActi...')
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#9 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#11 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#12 C:\wamp\www\dev.ppeditor\index.php(1): require_once('C:\wamp\www\dev...')
#13 {main}
2014-02-06T02:43:10-08:00 ERR (3): Application_Model_DbTable_Documents::sendRequet - exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:228
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(228): PDOStatement->execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(211): Zend_Db_Table_Abstract->insert(Array)
#7 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(657): Application_Model_DbTable_Documents->sendRequet(Array)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:234
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(211): Zend_Db_Table_Abstract->insert(Array)
#6 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(657): Application_Model_DbTable_Documents->sendRequet(Array)
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#14 {main}
2014-02-06T03:07:47-08:00 ERR (3): Application_Model_DbTable_Documents::sendRequet - exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:228
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(228): PDOStatement->execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(211): Zend_Db_Table_Abstract->insert(Array)
#7 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(657): Application_Model_DbTable_Documents->sendRequet(Array)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:234
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(211): Zend_Db_Table_Abstract->insert(Array)
#6 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(657): Application_Model_DbTable_Documents->sendRequet(Array)
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(30): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#14 {main}
%2014-02-11T01:49:42-08:00% %3Name% (%3%): %Application_Model_DbTable_Documents::sendRequet - exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#7 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#8 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#15 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#16 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#7 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}%
%2014-02-11T01:50:28-08:00% %3Name% (%3%): %Application_Model_DbTable_Documents::sendRequet - exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#7 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#8 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#15 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#16 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#7 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}%
%2014-02-11T01:50:35-08:00% %3Name% (%3%): %Application_Model_DbTable_Documents::sendRequet - exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#7 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#8 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#15 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#16 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#7 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}%
%2014-02-11T01:52:34-08:00% %3Name% (%3%): %Application_Model_DbTable_Documents::sendRequet - exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#7 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#8 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#15 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#16 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Status' in 'field list'' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('INSERT INTO `pp...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'INSERT INTO `pp...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('INSERT INTO `pp...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO `pp...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(575): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO `pp...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Table\Abstract.php(1075): Zend_Db_Adapter_Abstract->insert('pp_messages', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Documents.php(208): Zend_Db_Table_Abstract->insert(Array)
#7 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(600): Application_Model_DbTable_Documents->sendRequet(Array)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}%
%2014-02-16T21:47:35-08:00% %3Name% (%3%): %AuthorController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Missing To addresses' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:173
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(337): Zend_Mail_Transport_Sendmail->_prepareHeaders(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(804): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(652): AuthorController->_send('review', 'The  <a href="h...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->sendRequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendRequestActi...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-17T01:36:09-08:00% %3Name% (%3%): %AuthorController::documentsAction - exception 'PDOException' with message 'SQLSTATE[HY000]: General error: 2031 ' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:228
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(228): PDOStatement->execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Select.php(686): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(123): Zend_Db_Select->query(2)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(164): Zend_Paginator_Adapter_DbSelect->setRowCount(Object(Zend_Db_Table_Select))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(1060): Zend_Paginator_Adapter_DbSelect->count()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(740): Zend_Paginator->_calculatePageCount()
#9 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(86): Zend_Paginator->setItemCountPerPage('10')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->documentsAction()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('documentsAction')
#12 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#14 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#15 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#16 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#17 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[HY000]: General error: 2031 ' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:234
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Select.php(686): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#4 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(123): Zend_Db_Select->query(2)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(164): Zend_Paginator_Adapter_DbSelect->setRowCount(Object(Zend_Db_Table_Select))
#6 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(1060): Zend_Paginator_Adapter_DbSelect->count()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(740): Zend_Paginator->_calculatePageCount()
#8 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(86): Zend_Paginator->setItemCountPerPage('10')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->documentsAction()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('documentsAction')
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#15 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#16 {main}%
%2014-02-18T23:01:33-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(722): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(346): AdminController->_send('sendToTA', 'DPO, please ope...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->sendtotaAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendtotaAction')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-18T23:04:10-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(722): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(346): AdminController->_send('sendToTA', 'DPO, please ope...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->sendtotaAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('sendtotaAction')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-18T23:04:35-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(722): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(153): AdminController->_send('editRequest', 'playbook <b> Sh...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-18T23:04:56-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(722): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(153): AdminController->_send('editRequest', 'playbook <b> Sh...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-18T23:05:22-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(722): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(153): AdminController->_send('editRequest', 'playbook <b> Sh...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T04:22:00-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(721): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(152): AdminController->_send('editRequest', 'playbook <b> Sh...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T04:23:09-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(721): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(152): AdminController->_send('editRequest', 'playbook <b> Sh...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T04:23:23-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(721): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(152): AdminController->_send('editRequest', 'playbook <b> Sh...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T04:23:38-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(721): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(152): AdminController->_send('editRequest', 'playbook <b> Do...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T22:22:00-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(724): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(174): AdminController->_send('closed', '<p> Uploaded  S...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T22:23:58-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(724): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(174): AdminController->_send('closed', '<p> Uploaded  S...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-19T23:31:51-08:00% %3Name% (%3%): %Application_Model_DbTable_Requests::getAuthorDetail - exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'status.pp_requests' doesn't exist' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('SELECT `r`.`Nam...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('SELECT `r`.`Nam...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'SELECT `r`.`Nam...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('SELECT `r`.`Nam...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(360): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#6 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(145): Application_Model_DbTable_Requests->getAuthorDetail('41')
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#14 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'status.pp_requests' doesn't exist' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('SELECT `r`.`Nam...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'SELECT `r`.`Nam...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('SELECT `r`.`Nam...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(360): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(145): Application_Model_DbTable_Requests->getAuthorDetail('41')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#9 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#11 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#12 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#13 {main}%
%2014-02-19T23:33:32-08:00% %3Name% (%3%): %Application_Model_DbTable_Requests::getAuthorDetail - exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'status.pp_requests' doesn't exist' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('SELECT `r`.`Nam...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('SELECT `r`.`Nam...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'SELECT `r`.`Nam...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('SELECT `r`.`Nam...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(360): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#6 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(145): Application_Model_DbTable_Requests->getAuthorDetail('41')
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#14 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'status.pp_requests' doesn't exist' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('SELECT `r`.`Nam...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'SELECT `r`.`Nam...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('SELECT `r`.`Nam...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(360): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(145): Application_Model_DbTable_Requests->getAuthorDetail('41')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#9 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#11 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#12 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#13 {main}%
%2014-02-19T23:34:36-08:00% %3Name% (%3%): %Application_Model_DbTable_Requests::getAuthorDetail - exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'status.pp_requests' doesn't exist' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('SELECT `r`.`Nam...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('SELECT `r`.`Nam...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'SELECT `r`.`Nam...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('SELECT `r`.`Nam...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(360): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#6 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(145): Application_Model_DbTable_Requests->getAuthorDetail('41')
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#14 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'status.pp_requests' doesn't exist' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('SELECT `r`.`Nam...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'SELECT `r`.`Nam...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('SELECT `r`.`Nam...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(360): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(145): Application_Model_DbTable_Requests->getAuthorDetail('41')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#9 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#11 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#12 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#13 {main}%
%2014-02-20T04:45:26-08:00% %3Name% (%3%): %Application_Model_DbTable_Requests::updateAdminRequest - exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:58
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(58): PDO->prepare('UPDATE `pp_requ...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('UPDATE `pp_requ...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'UPDATE `pp_requ...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('UPDATE `pp_requ...')
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('UPDATE `pp_requ...', Array)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(634): Zend_Db_Adapter_Pdo_Abstract->query('UPDATE `pp_requ...', Array)
#6 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(299): Zend_Db_Adapter_Abstract->update('pp_requests', Array, 'RequestId = ')
#7 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(137): Application_Model_DbTable_Requests->updateAdminRequest(Array, NULL)
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#13 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#14 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#15 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:61
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(115): Zend_Db_Statement_Pdo->_prepare('UPDATE `pp_requ...')
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(184): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Pdo_Mysql), 'UPDATE `pp_requ...')
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(478): Zend_Db_Adapter_Pdo_Abstract->prepare('UPDATE `pp_requ...')
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query('UPDATE `pp_requ...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(634): Zend_Db_Adapter_Pdo_Abstract->query('UPDATE `pp_requ...', Array)
#5 C:\wamp\www\dev.ppeditor\application\models\DbTable\Requests.php(299): Zend_Db_Adapter_Abstract->update('pp_requests', Array, 'RequestId = ')
#6 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(137): Application_Model_DbTable_Requests->updateAdminRequest(Array, NULL)
#7 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#12 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#13 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#14 {main}%
%2014-02-20T04:46:07-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(732): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(182): AdminController->_send('closed', '<p> Uploaded  P...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-20T04:46:42-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(732): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(182): AdminController->_send('closed', '<p> Uploaded  S...', Array, Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-21T02:24:25-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(747): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(197): AdminController->_send('closed', 'Dear MTAdmin,<b...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-21T07:32:46-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(747): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(197): AdminController->_send('closed', 'Dear Dipa,<br><...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-21T07:33:44-08:00% %3Name% (%3%): %AdminController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Unable to send mail. mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:137
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(348): Zend_Mail_Transport_Sendmail->_sendMail()
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(747): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AdminController.php(197): AdminController->_send('closed', 'Dear Dipa,<br><...', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AdminController->updaterequestAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('updaterequestAc...')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-21T07:48:52-08:00% %3Name% (%3%): %AuthorController::_send - exception 'Zend_Mail_Transport_Exception' with message 'Missing To addresses' in C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Sendmail.php:173
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Mail\Transport\Abstract.php(337): Zend_Mail_Transport_Sendmail->_prepareHeaders(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Mail.php(1194): Zend_Mail_Transport_Abstract->send(Object(Zend_Mail))
#2 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(818): Zend_Mail->send()
#3 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(572): AuthorController->_send('authorToAdmin', 'afsdfsdafasfd', Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->messageAction()
#5 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('messageAction')
#6 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#9 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#10 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#11 {main}%
%2014-02-21T07:57:02-08:00% %3Name% (%3%): %AuthorController::documentsAction - exception 'PDOException' with message 'SQLSTATE[HY000]: General error: 2031 ' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:228
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php(228): PDOStatement->execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#4 C:\wamp\www\dev.ppeditor\library\Zend\Db\Select.php(686): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#5 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(123): Zend_Db_Select->query(2)
#6 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(164): Zend_Paginator_Adapter_DbSelect->setRowCount(Object(Zend_Db_Table_Select))
#7 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(1060): Zend_Paginator_Adapter_DbSelect->count()
#8 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(740): Zend_Paginator->_calculatePageCount()
#9 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(89): Zend_Paginator->setItemCountPerPage('10')
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->documentsAction()
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('documentsAction')
#12 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#14 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#15 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#16 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#17 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[HY000]: General error: 2031 ' in C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement\Pdo.php:234
Stack trace:
#0 C:\wamp\www\dev.ppeditor\library\Zend\Db\Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 C:\wamp\www\dev.ppeditor\library\Zend\Db\Adapter\Pdo\Abstract.php(238): Zend_Db_Adapter_Abstract->query(Object(Zend_Db_Table_Select), Array)
#3 C:\wamp\www\dev.ppeditor\library\Zend\Db\Select.php(686): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Table_Select))
#4 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(123): Zend_Db_Select->query(2)
#5 C:\wamp\www\dev.ppeditor\library\Zend\Paginator\Adapter\DbSelect.php(164): Zend_Paginator_Adapter_DbSelect->setRowCount(Object(Zend_Db_Table_Select))
#6 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(1060): Zend_Paginator_Adapter_DbSelect->count()
#7 C:\wamp\www\dev.ppeditor\library\Zend\Paginator.php(740): Zend_Paginator->_calculatePageCount()
#8 C:\wamp\www\dev.ppeditor\application\controllers\AuthorController.php(89): Zend_Paginator->setItemCountPerPage('10')
#9 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Action.php(516): AuthorController->documentsAction()
#10 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Dispatcher\Standard.php(295): Zend_Controller_Action->dispatch('documentsAction')
#11 C:\wamp\www\dev.ppeditor\library\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 C:\wamp\www\dev.ppeditor\library\Zend\Application\Bootstrap\Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 C:\wamp\www\dev.ppeditor\library\Zend\Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 C:\wamp\www\dev.ppeditor\public\index.php(36): Zend_Application->run()
#15 C:\wamp\www\dev.ppeditor\index.php(2): require_once('C:\wamp\www\dev...')
#16 {main}%
